### PR TITLE
learntheconcepts --> edits to oracles section

### PR DESCRIPTION
**Pull request by FelixNicolaeBucsa**

### CODE REVIEW

File: docs/learn_the_concepts/blockchains/oracles.md [hunk 1]
```diff
@@ -1,57 +1,67 @@
-Blockchains get their most significant positive features by being purposefully segregated from external systems. However, off-chain data cannot be accessed by blockchains or smart contracts directly, and these need such data to perform any contractual agreement between any party on the blockchain itself. This is known as the core of **the Oracle dilemma**. 
+Blockchains get their most significant positive features by being purposefully segregated from external systems. However, off-chain data cannot be accessed by blockchains or smart contracts directly, and these need such data to perform any contractual agreement between any party on the blockchain itself.
```
Comment (FelixNicolaeBucsa) [inline docs/learn_the_concepts/blockchains/oracles.md:2]: We could add a similar table to https://blokt.com/icos/chainlink-promising-blockchain-oracle

File: docs/learn_the_concepts/blockchains/oracles.md [hunk 2]
```diff
@@ -1,57 +1,67 @@
...
+!!! info
 
-A blockchain needs an additional piece of infrastructure, an **Oracle**, to connect the blockchain to the off-chain services in a secure manner. _A blockchain Oracle is a safe piece of middleware that makes it easier for blockchains to communicate with any off-chain system, such as data providers, online APIs, business backends, cloud providers, IoT devices, e-signatures, payment systems, other blockchains, and more_. This is, a blockchain Oracle is the element that links the blockchain to the external system, making it possible for smart contracts to operate on the basis of inputs and outputs from the real world. 
+    This is known as the core of **the oracle dilemma**. In fact,git blockchains are unable to draw data from or push data out to any external systems as part of their built-in functionality. A blockchain's isolation is precisely what makes it so safe and dependable since the network just needs to come to consensus on a very simple set of questions using information that has already been written in the ledger.
```
Comment (VijaySharma815) [inline docs/learn_the_concepts/blockchains/oracles.md]: In fact,git blockchains are unable to draw -> git blockchain?

Comment (FelixNicolaeBucsa) [reply docs/learn_the_concepts/blockchains/oracles.md]: Nope, my mistake

Comment (FelixNicolaeBucsa) [reply docs/learn_the_concepts/blockchains/oracles.md]: Gonna fix it

**Pull request merged**